### PR TITLE
Update for 1.17.4

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: _includes/manuals/1.17/1.2_release_notes.md [hunk 1]
```diff
@@ -259,8 +264,8 @@ Due to Ruby incompatibility, Debian/Jessie support was dropped in Foreman 1.17
 ### Upgrade warnings
 
 * When seeding default roles, permission check and auditiong is ignored ([#21011](http://projects.theforeman.org/issues/21011))
-* When additional filters were added to default role, the db:migrate fails ([#24697](http://projects.theforeman.org/issues/24697))
-* Array/hash values for Lookup Values turn into strings with escaped quotes ([#24640](http://projects.theforeman.org/issues/24640))
+
+The 1.17.4 is the last release for the 1.17 version. If you experience any problems, we advise to upgrade directly to 1.18.
```
Comment (mmoll) [inline _includes/manuals/1.17/1.2_release_notes.md]: Without "The ", please ;)
I'm unsure if it's really good to recommend the 1.18 update before 1.18.2 is out

Comment (ekohl) [reply _includes/manuals/1.17/1.2_release_notes.md]: You could also replace `the 1.17 version` with `the 1.17 series`

**Comment by xprazak2** [comment]
Updated

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**